components/eamxx/src/share/field/field_utils_impl.hpp

```diff
@@ -29,6 +29,8 @@ bool views_are_equal(const Field& f1, const Field& f2, const ekat::Comm* comm)
   f2.sync_to_host();
 
   // Reshape based on the rank, then loop over all entries.
+  // NOTE: because views_are_equal() is only used for testing, we generalize by
+  // always calling get_strided_view(), even if it could be contiguous
```

Comment:
I would add another line:
```
  // This allows to call this function on any type of field, including multi-slice subfiedls
```
Otherwise the user may wonder why we cannot use `get_view`...